src/components/button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Button = ({
+  children,
+  secondary,
+  loading,
+  disabled,
+  accessibilityLabel,
+  onClick = () => { },
+  to,
+  className
+}) => {
+  const buttonClassNames = classNames(
+    'button',
+    {
+      'button--secondary': secondary,
+      'cursor-not-allowed': disabled
```

Comment:
As you are declaring classes in CSS, you may as well do them all in one place. You're already adding `bg-gray-500` there